packages/core/src/config/config.ts

```diff
@@ -52,17 +53,18 @@ export class Config {
   }
   /**
    * Creates a domain object from config object
-   * @param {EthereumConfig | SubstrateConfig} config
+   * @param {EthereumConfig | SubstrateConfig | BitcoinConfig} config
    * @returns {Domain}
    */
-  private createDomain(config: EthereumConfig | SubstrateConfig): Domain {
+  private createDomain(config: EthereumConfig | SubstrateConfig | BitcoinConfig): Domain {
```

Comment:
can be simplified like

`SygmaDomainConfig = EthereumConfig | SubstrateConfig | BitcoinConfig`